src/app/login/login.component.ts

```diff
@@ -54,7 +54,7 @@ export class LoginComponent implements OnInit {
                     this.router.navigate([this.returnUrl]);
                 },
                 error => {
-                    this.alertService.createErrorAlert('invalid email or password provided', true);
+                    this.alertService.createErrorAlert('invalid email or password provided', false);
```

Comment:
Are you sure about this change? Because I don't see any changes made in alertService

Reply:
Yeah, if i leave on true, then the error message will stay if i go to another page instead, which is not wanted. With this change, as soon as i change the page, no alert will remain